bin/test

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use CodeIgniter\CLI\CLI;
+use Translations\Tests\AbstractTranslationTestCase;
+
+$phpCsFixer = 'vendor/bin/php-cs-fixer';
+$phpunit    = 'vendor/bin/phpunit';
+if (CLI::isWindows()) {
+    $phpCsFixer = strtr($phpCsFixer, '/', '\\');
+    $phpunit    = strtr($phpunit, '/', '\\');
+}
+
+CLI::write('Running `composer update`...', 'black', 'green');
+passthru('composer update');
+CLI::write();
+
+CLI::write('Fixing coding style...', 'black', 'green');
+passthru(
+    "{$phpCsFixer} fix --verbose --diff"
+);
```

Comment:
Can make this one line. Ansi output does not look good on Windows so we can leave it as is.
```suggestion
passthru("{$phpCsFixer} fix --verbose --diff");
```